### PR TITLE
making things right with creating new order (I hope so)

### PR DESCRIPTION
**Pull request by kjendrzyca**
A little reorganized creation of new order. After creating new order user is redirected to the dashboard, we can redirect him to the order details later on.

### CODE REVIEW

File: src/components/app.jsx [hunk 1]
```diff
@@ -47,6 +48,7 @@ DOM.render(
             <Route component={App} path="/">
                 <IndexRoute component={Dashboard} />
                 <Route component={NewOrderForm} path="newOrder" />
+                <Route component={OrderDetails} path="order" />
```
Comment (kjendrzyca) [inline src/components/app.jsx:51]: WIP

File: src/components/orderDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (kjendrzyca) [inline src/components/orderDetails.jsx:1]: WIP

File: src/dateManipulation.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+exports.mapHourToDate = hour => {
+    const array = hour.split(':');
```
Comment (tarnas14) [inline src/dateManipulation.js:2]: `const [hours, minutes] = hour.split(':');`

Comment (kjendrzyca) [inline src/dateManipulation.js:2]: tried it, doesn't work with clean node

Comment (tarnas14) [inline src/dateManipulation.js:2]: crap, right, it's node - I was even there when you tried it LOL

Comment (Wojtasik) [inline src/dateManipulation.js:2]: If we want, we can turn this feature on - [check it](http://stackoverflow.com/a/17379714/1777334)

Comment (kjendrzyca) [inline src/dateManipulation.js:2]: it doesn't work with --harmony_destructuring flag enabled :(
edit:
I was wrong, it works fine, will change that with next commit

**Pull request merged**